### PR TITLE
Milestone 4: add Basic UI

### PR DESCRIPTION
**Pull request by Efiamotu-1**
## Implemented Requirements

In this PR, the following were implemented:-

>- Created app file to present functionalities
>- Created main file to execute the app

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+    @people = []
+    @book = []
+    @rentals = []
+  end
+
+  def run
+    puts 'Welcome to school library App!'
+    puts ''
+    puts ''
+    puts 'Please put an option by entering a number!'
```
Comment (tufoinnkuo10) [inline app.rb:18]: - [optional] you can as well add which output it should give if user selects and option which is not found on the list. nice work

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Efiamotu-1 

Your project is complete! Great job for nailing it right on the first attempt 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- App works as expected
- all options available and create succesfully
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

tufoin

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 1 above)
Comment (Efiamotu-1) [reply app.rb:18]: Thank you @tufoinnkuo10 for your review and suggestion.

Comment (Efiamotu-1) [reply app.rb:18]: > Thank you @tufoinnkuo10 for your review and suggestion. i however provided for the scenerio in [these lines ](https://github.com/Efiamotu-1/OOP-School-Library/blob/ec1e4fb3d696ac44b04351dd8fa3e7c571d7c769/app.rb#L53-L55)

**Pull request merged**